catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/ws/JsonRpc.java

```diff
@@ -57,7 +57,11 @@ public static Error error(int code, String message) {
     return new Error(code, message);
   }
 
-  public static Error error(int code, String message, Object data) {
+  public static Error invalidParams(String message, Object params) {
+    return error(INVALID_PARAMS, String.format("Invalid params: %s", message), params);
```

Comment:
Are you thinking I should get rid of the prefix entirely? To use a usage of this as an example, instead of `Invalid params: params not list` it would just be `params not list`. (I think in looking more closely at this, I'm going to replace `params` with `parameters` in all error messages)

Reply:
`params` is the key used in the JSON RPC spec